src/ucp/rndv/rndv_ppln.c

```diff
@@ -305,7 +314,7 @@ ucp_proto_t ucp_rndv_send_ppln_proto = {
         [UCP_PROTO_RNDV_PPLN_STAGE_SEND] = ucp_proto_rndv_ppln_progress,
         [UCP_PROTO_RNDV_PPLN_STAGE_ACK]  = ucp_proto_rndv_send_ppln_atp_progress,
     },
-    .abort    = ucp_proto_abort_fatal_not_implemented,
+    .abort    = ucp_proto_rndv_ppln_abort,
```

Comment:
Maybe it would be better to leave that aborts pointers set to `ucp_proto_abort_fatal_not_implemented` so when it is called, user will  get hard error instead of half-ready abortion implementation without memory invalidation?

Reply:
For now I would keep it for 2 reasons:
1. As discussed on the meeting, we don't want to crash the application in case of remote peer failure.
2. Current protocol selection logic explicitly requires that abort function pointer is set to a different function:
```C
void ucp_proto_select_add_proto
...
    if (init_params->ep_config_key->err_mode != UCP_ERR_HANDLING_MODE_NONE) {
        ucs_assertv(ucp_protocols[proto_id]->abort !=
                            ucp_proto_abort_fatal_not_implemented,
                    "error handling is enabled, but %s doesn't support abort()",
                    proto_name);
    }
```

Reply:
maybe add warning message to `ucp_proto_rndv_ppln_abort` and `ucp_proto_rndv_put_mtype_abort`? BTW, they are 100% identical, so can be replaced with one common function, for example `ucp_proto_ppln_abort `

Reply:
Ok for warning message
Ok for common function